stacks/serverless/dovetail-counts-lambda.yml

```diff
@@ -0,0 +1,108 @@
+# stacks/serverless/dovetail-counts-lambda.yml
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Dovetail counts lambda function
+Parameters:
+  OpsWarnMessagesSnsTopicArn:
+    Type: String
+  OpsErrorMessagesSnsTopicArn:
+    Type: String
+  CodeS3Bucket:
+    Type: String
+  CodeS3ObjectVersion:
+    Type: String
+  EnvironmentType:
+    Type: String
+  EnvironmentTypeAbbreviation:
+    Type: String
+  VPC:
+    Type: "AWS::EC2::VPC::Id"
+  VPCSubnet1:
+    Type: "AWS::EC2::Subnet::Id"
+  VPCSubnet2:
+    Type: "AWS::EC2::Subnet::Id"
+  VPCSubnet3:
+    Type: "AWS::EC2::Subnet::Id"
+  KinesisIn:
+    Type: AWS::SSM::Parameter::Value<String>
+  KinesisOut:
+    Type: AWS::SSM::Parameter::Value<String>
+  RedisUrl:
+    Type: AWS::SSM::Parameter::Value<String>
+  S3Bucket:
+    Type: AWS::SSM::Parameter::Value<String>
+  S3Prefix:
+    Type: AWS::SSM::Parameter::Value<String>
+Resources:
+  DovetailCountsLambdaExecutionIAMRole:
+    Type: "AWS::IAM::Role"
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+            - Effect: "Allow"
+              Principal:
+                Service:
+                  - "lambda.amazonaws.com"
+              Action:
+                - "sts:AssumeRole"
+      Path: "/"
+      Policies:
+        - PolicyName: KinesisWritePolicy
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "kinesis:DescribeStream"
+                  - "kinesis:PutRecord"
+                  - "kinesis:PutRecords"
+                Resource:
+                  - !Ref KinesisOut
+      ManagedPolicyArns:
+        - "arn:aws:iam::aws:policy/AmazonS3ReadOnlyAccess"
+        - "arn:aws:iam::aws:policy/service-role/AWSLambdaKinesisExecutionRole"
+        - "arn:aws:iam::aws:policy/service-role/AWSLambdaVPCAccessExecutionRole"
+  DovetailCountsLambdaFunction:
+    Type: "AWS::Lambda::Function"
+    Properties:
+      Code:
+        S3Bucket: !Ref CodeS3Bucket
+        S3Key: lambda/PRX-dovetail-counts-lambda.zip
+        S3ObjectVersion: !Ref CodeS3ObjectVersion
+      Description: Dovetail bytes sent counter
+      Environment:
+        Variables:
+          KINESIS_OUT: !Ref KinesisOut
+          REDIS_URL: !Ref RedisUrl
+          S3_BUCKET: !Ref S3Bucket
+          S3_PREFIX: !Ref S3Prefix
+      Handler: index.handler
+      MemorySize: 128
+      Role: !GetAtt DovetailCountsLambdaExecutionIAMRole.Arn
+      Runtime: nodejs8.10
+      Timeout: 10
+      VpcConfig:
+        SecurityGroupIds: !GetAtt VPC.DefaultSecurityGroup
```

Comment:
Found some forum post with something like this in there ... but not sure it'll actually work.